src/bilder/components/concourse/models.py

```diff
@@ -1464,6 +1464,14 @@ def concourse_env(self) -> Dict[str, str]:
 class ConcourseWorkerConfig(ConcourseBaseConfig):
     _node_type: str = "worker"
     user: str = "root"
+    ## TODO: MAD 02162022 Correct this deploy_directory reference.
+    ## This 'works' but it isn't correct. It will always reference the default value for
+    ## deploy_directory, rather than any that is overridden in the actual instantiation, whereever
+    ## that may be. Be aware.
+    resource_types_directory: Path = (
+        ConcourseBaseConfig().deploy_directory / "resource-types"
```

Comment:
The problem with this approach is that we want to make this relative to the configured deploy directory. With this it will always use the default value from the parent class, rather than whatever customized location is passed into the instance creation of this class.